<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>461</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="FreeSlot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Free Slot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TargetSlot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Slot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BusySlot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Busy Slot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Obstacle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Obstacle</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
